Makefile

```diff
@@ -35,3 +41,89 @@ build-dependencies-axolotl:
 clean:
 	rm -f axolotl
 	rm -rf axolotl-web/dist
+
+check_platform_arm64:
+	@echo "Building Axolotl for Mobian (aarch64/amd64)"
```

Comment:
Is this really mobian-specific, or could we perhaps rename it to simply "Building Axolotl for arm64"?

Reply:
It can be changed into "Debian arm64/aarch64 systems"... no problem!